plugins/woocommerce/changelog/fix-add-to-cart-button-role

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Add role='button' to product grid Add to Cart buttons when they act as a button
```

Comment:
Nitpick, but maybe we should mention it's about "`anchors` that act as buttons".

```suggestion
Add role='button' to product grid Add to Cart anchors when they act as a button
```

Reply:
Good call, merged!